elb_automation/meaoelb/config.py

```diff
@@ -3,6 +3,7 @@
 from pygments.lexers import JsonLexer
 from pygments.formatters import TerminalFormatter
 import re
+from meaoelb.templates import *
```

Comment:
NIT: this is an [antipattern](https://docs.quantifiedcode.com/python-anti-patterns/maintainability/from_module_import_all_used.html)